goRename.py

```diff
@@ -10,7 +10,7 @@
 
 # TODO: review & clean
 
-import sublime, sublime_plugin, subprocess, time, re, os, subprocess, sys, time, hashlib
+import sublime, sublime_plugin, subprocess, time, re, shlex, os, subprocess, sys, time, hashlib
```

Comment:
Thanks mkleina !

it would be great if this conditional import 
```python
try:
    from shlex import quote as cmd_quote
except ImportError:
    from pipes import quote as cmd_quote
```
would be use instead of 
```python
import ... shlex
```
to provide better compatibility